src/components/pages/PendingApplications/ApplicationModal.js

```diff
@@ -170,9 +170,6 @@ const ApplicationModal = ({
                 <b>Location:</b> {currentApplication.city},{' '}
                 {currentApplication.state} {currentApplication.country}
               </List.Item>
-              <List.Item>
-                <b>Experience Level:</b> {currentApplication.experience_level}
-              </List.Item>
```

Comment:
Thanks for catching this bug that was clearly missed earlier! I'm also thrilled that it's such an easy fix!

Reply:
just following up on my comment because it's not being recorded as a review.